mopro-ffi/tests/test_generated_bindings.rs

```diff
@@ -1,3 +1,21 @@
+#[cfg(feature = "gpu-benchmarks")]
+uniffi::build_foreign_language_testcases!(
+    // "tests/bindings/test_mopro_gen_benchmarks_report.swift", // Uncomment to generate benchmarks report, takes nearly 10 minutes
```

Comment:
Shouldn't need to do uncomment if gated by feature flag afaik

Reply:
Same here

https://github.com/oskarth/mopro/pull/56#discussion_r1473756751